implementations/rust/ockam/ockam_node/src/node.rs

```diff
@@ -10,7 +10,7 @@ pub struct NullWorker;
 
 impl NullWorker {
     /// Create and register a new NullWorker context
-    pub(crate) fn new(rt: Arc<Runtime>, addr: &Address, tx: Sender<NodeMessage>) -> Context {
+    pub fn new(rt: Arc<Runtime>, addr: &Address, tx: Sender<NodeMessage>) -> Context {
```

Comment:
Is there purpose for such worker in public API?

Reply:
Reverting this, was an experiment